src/common.js

```diff
@@ -156,8 +156,7 @@ function applyPackages (id, packages) {
   const pkgName = getMatch(id, packages);
   if (pkgName) {
     const pkg = packages[pkgName];
-    if (pkg === null)
-
+    if (pkg === null) return;
```

Comment:
Is this fixing an existing bug? Looks like we just had an if statement without a block afterwards?

Reply:
Yup! The validation was never applying :astonished: